ImperatorToCk3/Source/Configuration/Configuration.cpp

```diff
@@ -25,32 +25,58 @@ Configuration::Configuration(std::istream& theStream)
 
 void Configuration::registerKeys()
 {
-	registerKeyword("configuration", [this](const std::string& unused, std::istream& theStream) { details = ConfigurationDetails(theStream); });
+	registerKeyword("SaveGame", [this](const std::string& unused, std::istream& theStream) {
+		const commonItems::singleString path(theStream);
+		SaveGamePath = path.getString();
+		Log(LogLevel::Info) << "Save Game set to: " << SaveGamePath;
+		});
+	registerKeyword("ImperatorDirectory", [this](const std::string& unused, std::istream& theStream) {
+		const commonItems::singleString path(theStream);
+		ImperatorPath = path.getString();
+		});
+	registerKeyword("ImperatorModsDirectory", [this](const std::string& unused, std::istream& theStream) {
+		const commonItems::singleString path(theStream);
+		ImperatorModsPath = path.getString();
+		});
+	registerKeyword("CK3directory", [this](const std::string& unused, std::istream& theStream) {
+		const commonItems::singleString path(theStream);
+		Ck3Path = path.getString();
+		});
+	registerKeyword("output_name", [this](const std::string& unused, std::istream& theStream) {
+		const commonItems::singleString nameStr(theStream);
+		outputName = nameStr.getString();
+		Log(LogLevel::Info) << "Output name set to: " << outputName;
+		});
+	registerKeyword("ImperatorDeJure", [this](const std::string& unused, std::istream& theStream) {
```

Comment:
Fronter will serve strings, not ints, for all configuration options. You need singleString and stoi after.